client/Chats.jsx

```diff
@@ -0,0 +1,36 @@
+const React = require('react');
+
+const Chat = require('./Chat.jsx');
+const api = require('../server/api');
+
+// 1px border, 339px content
+
+class Chats extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+    this.state.chats = [];
+  }
+
+  componentDidMount() {
+    fetch(`${api}/2`)
```

Comment:
This actually worked? when I looked at what was required for api the file had a string '/api/chats'. I thought you also needed the root of the api (like localhost) to get it to fetch.

Reply:
There is a one-line file called. api.js that says 
module.exports = '/api/chats';

If I change that file, I can update both the client and the server api endpoints.

Reply:
Did not know. I want to try it out now.